src/GraphBLAS-sharp/Algorithms/BFS.fs

```diff
@@ -3,23 +3,27 @@ namespace GraphBLAS.FSharp.Algorithms
 open GraphBLAS.FSharp.Predefined
 open GraphBLAS.FSharp.Helpers
 open GraphBLAS.FSharp
+open Brahma.FSharp.OpenCL.WorkflowBuilder.Basic
+open Brahma.FSharp.OpenCL.WorkflowBuilder.Evaluation
 
 [<AutoOpen>]
 module BFS =
-    let levelBFS (matrix: Matrix<bool>) (source: int) : Vector<int> =
+    let levelBFS (matrix: Matrix<bool>) (source: int) : OpenCLEvaluation<Vector<int>> =
```

Comment:
Can it be naturally generalized to multisource BFS?

Reply:
Yes, but at the moment we still won't be able to test it. These algorithms are given as an example of use and may not be completely correct.

Reply:
Ok.